libtensor/core/impl/xm_allocator.h

```diff
@@ -1,234 +1,223 @@
 #ifndef LIBTENSOR_XM_ALLOCATOR_H
```

Comment:
Sorry to be picky here, but can you turn clang-format off when committing this file.

Reply:
Ahh... sorry 😦